docs/source/noise/index.rst

```diff
@@ -27,7 +28,7 @@ Pseudopeople can add two broad categories of noise to the datasets it generates:
 
 #. **Row-based noise:** Errors in the inclusion or exclusion of entire rows of
    data, such as duplication or omission
-#. **Column-based noise:** Errors in data entry for individual fields within a
+#. **Column-based noise:** Errors in data entry for an individual field within a
```

Comment:
I think I'm going to revise this further in my PR, but it's fine for now.

Reply:
Yeah, I think we need to edit these definitions, but we can do it in a later PR. It occurred to me that in addition to inclusion or exclusion of rows, we could in the future have row-based noise that works similarly to column-based noise, where we select some rows at random or based on certain criteria, and then noise some of the fields in those specific rows. So probably the definitions for row-based and column-based noise should be more symmetrical. But again, that's for a later PR, just writing my thoughts here.